index.css

```diff
@@ -97,18 +152,60 @@ nav {
   margin: 4px 24px;
 }
 
+.vertical-links-container {
+  display: none;
+  grid-template-columns: 1fr;
+  justify-content: center;
+  align-items: center;
+  width: fit-content;
+  padding: 0 12px;
+  margin-right: 87px;
+  height: fit-content;
+}
+
 .links-container {
   display: flex;
   padding: 4px 12px;
 }
 
-.links-container a {
-  height: 17px;
-  color: white;
-  text-decoration: none;
-  align-items: flex-start;
-  padding: 0.01rem 8px;
-  list-style-type: none;
+.links-container li a img {
+  margin: auto 10.5px;
+}
+
+@media screen and (min-width: 768px) {
+  .main {
+    min-height: 798px;
+    background-image: url("./assets/Group-large.png");
+    background-position-x: 101%;
+    background-position-y: 100%;
+    justify-content: left;
+  }
+
+  .home-section {
+    max-width: 566px;
+  }
+
+  .vertical-links-container {
+    display: grid;
+  }
+
+  .section-title {
+    margin: 0.001px;
+  }
+
+  .info-title {
+    font-size: 32px;
+    margin: 12px auto;
+  }
+
+  .info {
+    font-size: 20px;
+    margin: 12px auto;
+  }
+
+  .links-container {
+    display: none;
+  }
 }
 
 .section-container {
```

Comment:
-  You did a great job by implementing some required changes given by the previous reviewer however you still have the issue with the background color I suggest you change the background to  `background: #FFFFFF;` in that section shown below It will make your design to stick to the Figma design as required in the project 
 
```
.section-container {
  background: #e5e5e5;
  min-height: 100vh;
  display: grid;
  align-items: center;
  justify-content: center;
}

```
Figma design 

![image](https://user-images.githubusercontent.com/95297251/206210256-6ad3ec63-e10e-49e5-bfca-5518f1d33505.png)

Your design

![image](https://user-images.githubusercontent.com/95297251/206210370-121a80a2-1d4c-40c1-8dd8-6a782883dae8.png)